client/verta/verta/endpoint/_build.py

```diff
@@ -1,57 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
This file is effectively moved to `client/verta/verta/endpoint/build/_build.py`, but it's pretty different so that new file is worth its own review